packages/vector_graphics_compiler/bin/vector_graphics_compiler.dart

```diff
@@ -76,7 +77,15 @@ Future<void> main(List<String> args) async {
   final String xml = File(inputFilePath).readAsStringSync();
   final File outputFile =
       File(results['output'] as String? ?? '$inputFilePath.vg');
-  final Uint8List bytes = await encodeSvg(xml, args[0]);
+
+  bool maskingOptimizerEnabled = true;
+
+  if (results['optimize-masks'] == false) {
+    maskingOptimizerEnabled = false;
+  }
```

Comment:
`final bool maskingOptimizerEnabled = results['optimize-masks'] ?? true;`

Reply:
vscode argues that ```results['optimize-masks']``` is dynamic, when I tried this, but it doesn't complain when I do an if statement.